### DISCUSSION TITLE
Row selection plugin shows no check boxes

### DISCUSSION THREAD

**Discussion by RuWi2210** [grid-js/gridjs#1445] [Q&A]
I am trying to make a table with a checkbox column as illustrated in [this example](https://gridjs.io/docs/plugins/selection/row-selection).
It is a part of a project that i am writing in python using [Flask](https://flask.palletsprojects.com/en/3.0.x/).

However, when i render the HTML file the table shows up with just an empty column where the checkboxes should be.
I have created a minimal example to illustrate the issue:

```
<!doctype html>
<html>
  <head>
    <!-- Table styling -->
    <link rel="stylesheet" href="https://unpkg.com/gridjs/dist/theme/mermaid.min.css"/>
  </head>

  <body>
        <div id="fab_four_table"></div>

    <!-- Import gridjs and selection plugin -->
    <script src="https://unpkg.com/gridjs/dist/gridjs.umd.js"></script>
    <script src="https://unpkg.com/gridjs/plugins/selection/dist/selection.umd.js"></script>
    
    <script>
    //   Define table columns and data
      new gridjs.Grid({
        columns: [
          { id: 'Checkbox', name: 'Select', plugin: {component: gridjs.plugins.selection.RowSelection}},
          { id: 'first_name', name: 'First Name' },
          { id: 'last_name', name: 'Last Name' },
          { id: 'instrument', name: 'Instrument'}
        ],
        
        data: [
            {
                first_name: 'Paul',
                last_name: 'McCartney',
                instrument: 'Bass'
            },
            {
                first_name: 'John',
                last_name: 'Lennon',
                instrument: 'Guitar/Piano'
            },
            {
                first_name: 'Ringo',
                last_name: 'Starr',
                instrument: 'Drums'
            },
            {
                first_name: 'Geroge',
                last_name: 'Harrison',
                instrument: 'Guitar'
            }
        ],
        width: 1000
      }).render(document.getElementById('fab_four_table'));
    </script>
  </body>
</html>
```

![image](https://github.com/grid-js/gridjs/assets/112623127/393c58d3-260d-48d9-9509-301ad802dcff)

I have noticed, that when i "inspect" the pages, the checkbox cells in the working example from the documentation looks like this:
![image](https://github.com/grid-js/gridjs/assets/112623127/fb27c45f-7248-4f65-8467-fcf2f4425e40)

Whereas in the table that i have rendered the checkbox cell looks like this:
![image](https://github.com/grid-js/gridjs/assets/112623127/9d7f923c-36a1-45d4-a4cf-c1cff64d1f28)

I.e. the cell in my table is missing an input sub-element, but i cannot figure out why.

Any help i greatly appreciated!

**Comment by IzerkT** [comment]
Hello, I have same issue. Did you find any solution?

**Reply by RuWi2210** [reply]
No, unfortunately i have not found a solution yet.

**Comment by slinaresperalta** [comment]
I also have the same exact issue.

**Reply by choudharypranay** [reply]
I found a workaround. This is broken in 6.2.0 version. So to revert to previous working version use the following:
    <script src="https://unpkg.com/gridjs@6.1.1/dist/gridjs.production.min.js"></script>
    <script src="https://unpkg.com/gridjs@6.1.1/plugins/selection/dist/selection.umd.js"></script>
    <link href="https://unpkg.com/gridjs@6.1.1/dist/theme/mermaid.min.css" rel="stylesheet" />

Note that 7 issues were fixed in version 6.2.0 (release notes: https://github.com/grid-js/gridjs/releases/tag/6.2.0) which might be another problem!
